Rules.mk

```diff
@@ -66,7 +66,13 @@ include $(dir)/Rules.mk
 # -------------------- #
 #     core targets     #
 # -------------------- #
+PACKAGES_NOVENDOR := $(shell go list github.com/ipfs/go-ipfs/... | grep -v /Godeps/)
 
+megacheck:
+	@go get honnef.co/go/tools/cmd/megacheck
+	@for pkg in ${PACKAGES_NOVENDOR}; do megacheck "$$pkg"; done
+
+.PHONY: megacheck
```

Comment:
Please add this target to `mk/golang.mk` instead and add comment about it to help print section.